clippy_lints/src/doc/mod.rs

```diff
@@ -373,11 +399,22 @@ impl<'tcx> LateLintPass<'tcx> for Documentation {
         check_attrs(cx, &self.valid_idents, attrs);
     }
 
+    fn check_variant(&mut self, cx: &LateContext<'tcx>, variant: &'tcx hir::Variant<'tcx>) {
```

Comment:
This is something that should be in the changelog, too.

Reply:
it's auto-generated?

Reply:
I think they meant in your first comment of this PR.